src/oic/oic/message.py

```diff
@@ -727,19 +749,44 @@ def verify(self, **kwargs):
 
 
 class RefreshSessionRequest(Message):
-    c_param = {"id_token": SINGLE_REQUIRED_STRING,
-               "redirect_url": SINGLE_REQUIRED_STRING,
-               "state": SINGLE_REQUIRED_STRING}
+    c_param = {
+        "id_token": SINGLE_REQUIRED_STRING,
+        "redirect_url": SINGLE_REQUIRED_STRING,
+        "state": SINGLE_REQUIRED_STRING
+    }
+
+    def verify(self, **kwargs):
```

Comment:
This can be probably placed into a common metaclass as it is used in multiple messages.

Reply:
This is nothing new that I added. But if we're changing other stuff why not this ?!

Reply:
Hm, weird... appeared in the diff as a new code. Maybe due to conflicts?